packages/contracts-periphery/.solhintignore

```diff
@@ -1 +1,2 @@
 node_modules
+contracts/foundry-tests/*.t.sol
```

Comment:
I will fix this in a new PR and add more tests

Reply:
TY, but TBH I don't think it's a priority to lint/format the solidity test files, and it will result in an enormous diff that will be frustrating to you.

Definitely appreciate your contributions here though. One open item we need solidity help with is #2883. Feel free to find me in our discord if you have questions or other ideas.